web/src/components/storage/ProposalVolumes.jsx

```diff
@@ -109,30 +115,33 @@ const GeneralActions = ({ templates, onAdd, onReset }) => {
             key="reset"
             onClick={onReset}
           >
-            Reset to defaults
+            {/* TRANSLATORS: dropdown menu label */}
```

Comment:
Why do we need the comment between brackets? There are more cases below.

Reply:
Because this is a JSX code, not regular JS.  The usual JS comments do not work here, you would get syntax error.
With curly braces you switch to JS code and there you can use the usual JS comments.

See more details e.g. in https://wesbos.com/react-jsx-comments

Reply:
Ah right! Sorry, I did not realize it was inside a JSX block.